Probesys/bileto
# new: Post confidential messages

Related to https://github.com/Probesys/bileto/issues/10

Changes proposed in this pull request:

- Rename message.isPrivate to message.isConfidential
- Add a checkbox to mark a message as confidential

How to test the feature manually:

1. Open a ticket
2. Answer to the ticket by checking the confidential checkbox
3. Verify that the solution checkbox is disabled when confidential is checked (and also the other way)
4. Verify that the message has a special design

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
